spec/controllers/hyrax/admin/admin_sets_controller_spec.rb

```diff
@@ -202,10 +202,11 @@
           end
 
           it 'shows the new form' do
-            expect(Rails.logger).to receive(:error).with(/Failed to create admin set through valkyrie:/)
+            expect(Rails.logger).to receive(:error).with(/Failed to create admin set:/)
```

Comment:
Since this same message is used for both Valkyrie and ActiveFedora AND since it is now shows to end users in a flash message, it makes more sense to not include `'through valkyrie'` in the message.